README.md

```diff
@@ -6,7 +6,6 @@ Platform             | CI Status
 ---------------------|:---------
 Linux (Focal)        | [![Build Status](https://github.com/ros-industrial-consortium/tesseract/workflows/Focal-Build/badge.svg)](https://github.com/ros-industrial-consortium/tesseract/actions)
 Linux (Bionic)       | [![Build Status](https://github.com/ros-industrial-consortium/tesseract/workflows/Bionic-Build/badge.svg)](https://github.com/ros-industrial-consortium/tesseract/actions)
-Linux (Xenial)       | [![Build Status](https://github.com/ros-industrial-consortium/tesseract/workflows/Xenial-Build/badge.svg)](https://github.com/ros-industrial-consortium/tesseract/actions)
```

Comment:
Gasp!

Reply:
> These are pretty confusing, but I think there are correct. Not something the average user will touch anyway.

They definitely are. It was a lot of trial and error.